tools/pelican/content/ipynb/tutorial-installation.ipynb

```diff
@@ -0,0 +1,12866 @@
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"243.118207\" xlink:href=\"#ma0c9089e31\" y=\"483.098695\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"246.211333\" xlink:href=\"#ma0c9089e31\" y=\"484.213685\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"249.30446\" xlink:href=\"#ma0c9089e31\" y=\"481.355539\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"252.397586\" xlink:href=\"#ma0c9089e31\" y=\"483.960803\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"255.490713\" xlink:href=\"#ma0c9089e31\" y=\"483.052885\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"258.583839\" xlink:href=\"#ma0c9089e31\" y=\"485.226531\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"261.676965\" xlink:href=\"#ma0c9089e31\" y=\"489.758726\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"264.770092\" xlink:href=\"#ma0c9089e31\" y=\"491.38633\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"267.863218\" xlink:href=\"#ma0c9089e31\" y=\"493.696585\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"270.956345\" xlink:href=\"#ma0c9089e31\" y=\"495.06903\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"274.049471\" xlink:href=\"#ma0c9089e31\" y=\"497.510221\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"277.142597\" xlink:href=\"#ma0c9089e31\" y=\"498.873586\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"280.235724\" xlink:href=\"#ma0c9089e31\" y=\"500.151057\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"283.32885\" xlink:href=\"#ma0c9089e31\" y=\"502.409008\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"286.421976\" xlink:href=\"#ma0c9089e31\" y=\"502.290552\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"289.515103\" xlink:href=\"#ma0c9089e31\" y=\"502.035503\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"292.608229\" xlink:href=\"#ma0c9089e31\" y=\"503.997437\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"295.701356\" xlink:hre
```

Comment:
```suggestion
    "If you can see the plot, you have successfully install `oscovida`, and are ready to use it."
```

Reply:
Er... not sure if the suggestion is selecting the wrong line in the preview for you too, the suggestion should be to fix the typo `sucessfully` -> `successfully`.

Reply:
I have fixed that locally; thanks!